tests/components/samsungtv/test_config_flow.py

```diff
@@ -273,57 +282,67 @@ async def test_user_not_successful_2(hass: HomeAssistant, remotews: Mock):
 
 async def test_ssdp(hass: HomeAssistant, remote: Mock):
     """Test starting a flow from discovery."""
+    with patch(
+        "homeassistant.components.samsungtv.bridge.SamsungTVWSBridge.device_info",
+        return_value=MOCK_DEVICE_INFO,
+    ):
```

Comment:
The `remote` fixture is for legacy, and the `remotews` fixture is for websocket.  By patching  `homeassistant.components.samsungtv.bridge.SamsungTVWSBridge`, this is effectively converting the test from testing the legacy devices to testing websocket devices